crawl-ref/source/dat/des/altar/altar.des

```diff
@@ -294,6 +294,72 @@ x.G.x
 ..@..
 ENDMAP
 
+# Simple decision for the player on whether the altar is worth stepping on
+# or otherwise avoiding (digging etc) one or more traps
+NAME:  amcnicky_altar_trapped
+TAGS:  no_monster_gen
+DEPTH: D:3-, Vaults
+SUBST: p:~., q:~....., r:~.......
```

Comment:
First can I say thank you for the detailed feedback and for taking the time to review my vaults. It is fantastic to get such a useful set of guiding comments to learn from. I will be working through them here and commenting responses mainly to track my changes versus your review comments. I might suggest that you turn off notifications for this pull request to avoid getting spammed :) Once ready, as suggested, you may see the some of these vaults resubmitted in further separate pull requests.

Thanks again!

Reply:
Ok yes, I think for now I'll drop the trap vaults from this set. I may look into doing trap vaults which are more of a challenge in future, at which point perhaps they could share some weight with existing trap vaults like the one you have noted above to avoid increasing 'trap vault spawn rate', but for now dropping this one.